src/main/java/hudson/plugins/s3/S3BucketPublisher.java

```diff
@@ -66,6 +68,7 @@ public S3BucketPublisher(String profileName, List<Entry> entries, List<MetadataP
         this.userMetadata = userMetadata;
 
         this.dontWaitForConcurrentBuildCompletion = dontWaitForConcurrentBuildCompletion;
+        this.consoleLogLevel = Level.parse(consoleLogLevel);
```

Comment:
@Jimilian Please check my latest commit to see if that is a more acceptable solution.